docs/theme/versions.html

```diff
@@ -0,0 +1,31 @@
+<div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
+  <span class="rst-current-version" data-toggle="rst-current-version">
+    <span class="fa fa-book"> {{ _('You are viewing') }}</span>
+    v: {{ version_label }}
+    <span class="fa fa-caret-down"></span>
+  </span>
+  <div class="rst-other-versions">
+    {# Temporarily disabling version switching
+    <dl>
+      <dt>{{ _('Versions') }}</dt>
+      {% for version in version_list %}
+        <dd><a class="version" href="{{ version_url(version, pagename) }}">{{ version }}</a></dd>
+      {% endfor %}
+    </dl>
+    #}
+    <dl>
+      <dt>{{ _('Languages') }}</dt>
+      {% for code, language in translations %}
+        <dd><a class="version" href="{{ translation_url(code, pagename) }}">{{ language }}</a></dd>
+      {% endfor %}
+    </dl>
+  </div>
+  <script>
+    jQuery('.version').click((evt) => {
+      const hash = window.location.hash
+      const complete_url = evt.target.href + hash
+      window.location = complete_url
+      evt.preventDefault()
+    })
+  </script>
+</div>
```

Comment:
```suggestion
</div>

```
Adding new line to the end of file